src/main/java/org/schemaspy/output/dot/schemaspy/DotOrphanFormatter.java

```diff
@@ -41,10 +41,21 @@ public DotOrphanFormatter(DotConfig dotConfig) {
     }
 
     public void writeOrphan(Table table, PrintWriter dot) {
-        DotFormat format = new DotFormat(dotConfig, table.getName(), false);
-        dot.println(format.header());
-        DotNodeConfig nodeConfig = new DotNodeConfig(true, true);
-        dot.println(new DotNode(table, true, nodeConfig, dotConfig).toString());
+        writeOrphan(
+                dot,
+                new DotConfigHeader(dotConfig, table.getName(), false),
+                new DotNode(
+                        table,
+                        true,
+                        new DotNodeConfig(true, true),
+                        dotConfig
+                )
+        );
+    }
+
+    public void writeOrphan(PrintWriter dot, Header header, Node node) {
```

Comment:
Only public due to test usage?

Will it be called by "external" production code in the fututre?

Reply:
Replace, but that's a large piece of work and the method will look different in the end.